src/main/java/scala_maven/ScalaCompilerLoader.java

```diff
@@ -19,10 +19,10 @@
 import java.net.URL;
 import java.net.URLClassLoader;
 
-public class Scala3CompilerLoader extends URLClassLoader {
+public class ScalaCompilerLoader extends URLClassLoader {
```

Comment:
This class is, in large part, copied from the [lampefl/dotty](https://github.com/lampepfl/dotty/blob/96401b68a8bce4c125859806b0c24a1ffe3cbc1e/sbt-bridge/src/xsbt/CompilerClassLoader.java) code base wich is licensed under [Apache License 2.0](https://github.com/lampepfl/dotty/blob/master/LICENSE). So I think the "Dotty (https://dotty.epfl.ch) Copyright 2012-2020 EPFL Copyright 2012-2020 Lightbend, Inc" should be preserved.

Reply:
Thanks, I updated the header. It allow to see that the header was set by the formatter (I didn't configure it) and that the wrong license header is injected. It should be unlicenced. I'll do a dedicated commit (all file to update)